torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -479,6 +479,54 @@ class TORCH_CUDA_CU_API TensorView : public Val {
       bool best_effort = false,
       MaxPosCalculator* calc = nullptr);
 
+  //! Inline the computation of this tensor into the given consumer at the given
+  //! position. The position is relative to the consumer. If this
+  //! tensor is already inlined in a higher position with the same
+  //! consumer, then this call is a no-op. The actual position is
+  //! computed in the same way as inlineAt, except that computeWith
+  //! does not have the constraint of the persistent data-dependency pattern.
+  void computeWith(
+      TensorView* consumer,
+      int consumer_pos,
+      bool best_effort = false);
+
+  bool hasComputeWith() const {
+    return !getComputeWithConsumers().empty();
+  }
+
+  bool isComputedWith(const TensorView* consumer) const {
+    return std::find(
+               getComputeWithConsumers().begin(),
+               getComputeWithConsumers().end(),
+               consumer) != getComputeWithConsumers().end();
+  }
+
+  //! Return the tensor with which this tensor is computed. nullptr is
+  //! returned if no computeWith is set
```

Comment:
I think you meant the following?
```
  //! Return the tensors with which this tensor is computed. empty vector is
  //! returned if no computeWith is set
```

Reply:
Oh, thanks. The comment was originally written when `compute_with_consumers_` is just `TensorView*`. Will update the comment.